--- keystone-paste.icehouse.ini	2015-02-02 15:50:58.000000000 +0100
+++ keystone-paste.icehouse.extended.ini	2015-02-02 16:46:48.000000000 +0100
@@ -30,6 +30,9 @@ paste.filter_factory = keystone.contrib.
 [filter:crud_extension]
 paste.filter_factory = keystone.contrib.admin_crud:CrudExtension.factory
 
+[filter:crud_extension_profile]
+paste.filter_factory = keystone.contrib.user_profile:CrudExtension.factory
+
 [filter:ec2_extension]
 paste.filter_factory = keystone.contrib.ec2:Ec2Extension.factory
 
@@ -79,10 +82,10 @@ paste.app_factory = keystone.service:v3_
 paste.app_factory = keystone.service:admin_app_factory
 
 [pipeline:public_api]
-pipeline = sizelimit url_normalize build_auth_context token_auth admin_token_auth xml_body_v2 json_body ec2_extension user_crud_extension public_service
+pipeline = sizelimit url_normalize build_auth_context token_auth admin_token_auth xml_body_v2 json_body ec2_extension user_crud_extension crud_extension_profile public_service
 
 [pipeline:admin_api]
-pipeline = sizelimit url_normalize build_auth_context token_auth admin_token_auth xml_body_v2 json_body ec2_extension s3_extension crud_extension admin_service
+pipeline = sizelimit url_normalize build_auth_context token_auth admin_token_auth xml_body_v2 json_body ec2_extension s3_extension crud_extension crud_extension_profile admin_service
 
 [pipeline:api_v3]
 pipeline = sizelimit url_normalize build_auth_context token_auth admin_token_auth xml_body_v3 json_body ec2_extension_v3 s3_extension simple_cert_extension service_v3
